libutils/string_lib.h

```diff
@@ -322,28 +322,6 @@ void CanonifyNameInPlace(char *str);
 bool StringMatchesOption(
     const char *supplied, const char *longopt, const char *shortopt);
 
-/**
```

Comment:
Please remove the `#include <seq.h>` from this file now that it's not needed anymore.